### ISSUE TITLE
NeoScore not working in python3.11

### ISSUE THREAD

**Issue by charlesneimog** [DigiScore/neoscore#84]
Running this: 
``` python3
from neoscore.common import *
neoscore.setup()
Text(ORIGIN, None, "Hello, neoscore!")
neoscore.show()
```
I get this:
```bash
Traceback (most recent call last):
  File "/home/neimog/Documents/Eco/sax_templates.py", line 1, in <module>
    from neoscore.common import *
  File "/home/neimog/Documents/OM#/temp-files/OM-py-env/lib64/python3.11/site-packages/neoscore/common.py", line 1, in <module>
    from neoscore.core import neoscore
  File "/home/neimog/Documents/OM#/temp-files/OM-py-env/lib64/python3.11/site-packages/neoscore/core/neoscore.py", line 18, in <module>
    from neoscore.core.paper import A4, Paper
  File "/home/neimog/Documents/OM#/temp-files/OM-py-env/lib64/python3.11/site-packages/neoscore/core/paper.py", line 9, in <module>
    @dataclass(frozen=True)
     ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/dataclasses.py", line 1211, in wrap
    return _process_class(cls, init, repr, eq, order, unsafe_hash,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/dataclasses.py", line 959, in _process_class
    cls_fields.append(_get_field(cls, name, type, kw_only))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/dataclasses.py", line 816, in _get_field
    raise ValueError(f'mutable default {type(f.default)} for field '
ValueError: mutable default <class 'neoscore.core.units.Unit'> for field margin_top is not allowed: use default_factory
``

**Comment by ajyoon** [comment]
Looks like [3.11 added a breaking change around this](https://docs.python.org/3/whatsnew/3.11.html#dataclasses). Not a complicated fix, I should be able to address in the next day or two, assuming there aren't other 3.11 breaking changes hiding.

**Comment by ajyoon** [comment]
Should be fixed now in neoscore 0.1.10, which I just published. Thanks for the report!

**Comment by charlesneimog** [comment]
Thanks!
